tests/components/smartthings/snapshots/test_switch.ambr

```diff
@@ -1033,50 +563,3 @@
     'state': 'off',
   })
 # ---
-# name: test_all_entities[vd_stv_2017_k][switch.tv_samsung_8_series_49-entry]
```

Comment:
Why are the snapshot modified if no changed test is using snapshots?

Reply:
Because these aren't created anymore if you didn't have them before